chandra_aca/drift.py

```diff
@@ -195,6 +197,14 @@ def get_fid_offset(time: CxoTimeLike, t_ccd: float) -> tuple:
     See https://github.com/sot/fid_drift_mon/blob/master/fid_offset_coeff.ipynb
     """
 
+    # Clip the time to the minimum time in the drift model
+    # Handle scalar and array cases
+    if np.isscalar(time):
+        time = np.max([CxoTime(time).secs, CxoTime("2012:001:12:00:00").secs])
```

Comment:
@taldcroft You'd mentioned that this function just expected a scalar so it would only need to handle the scalar case, but I had some validation use cases that took advantage of the fact that the "math" in the AcaDriftModel also works fine with np.arrays.  So I handled both cases here and updated the docstring to indicate this function will take scalars or arrays.

Reply:
I think this can be just below, which works for scalar and array inputs.
```
time = CxoTime(time).secs.clip(CxoTime("...").secs, None)
```